velox/exec/prefixsort/PrefixSortEncoder.h

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <cstdint>
+#include <functional>
+#include <memory>
+
+#include "velox/common/base/BitUtil.h"
+#include "velox/common/base/Exceptions.h"
+#include "velox/common/base/SimdUtil.h"
+
+namespace facebook::velox::exec::prefixsort {
+
+/// Provides encode/decode methods for PrefixSort.
+class PrefixSortEncoder {
+ public:
+  /// 1.Only int64_t is supported now.
```

Comment:
nit: add space after 1. and 2.

> Encoding should be compatible

Encoding is compatible ...

Reply:
> Encoding is compatible ...

I'm just a liitle curious about the usage of modal particles here. I think using 'is' only describes the current state, and 'should' can prompt other coders to understand the constraints if some want to add new type.

Reply:
Got it , thank a lot for explaining the reason~